pkg/operator/controller/prometheus.go

```diff
@@ -206,9 +211,10 @@ func getAlertRules() []promv1.Rule {
 				"runbook_url": runbookURLBasePath + "CDIDataImportCronOutdated",
 			},
 			map[string]string{
-				severityAlertLabelKey:  "info",
-				partOfAlertLabelKey:    partOfAlertLabelValue,
-				componentAlertLabelKey: componentAlertLabelValue,
+				severityAlertLabelKey:     "info",
+				healthImpactAlertLabelKey: "warning",
```

Comment:
@assafad Same as above, There is a loss of functionality and I think we should consider making this critical impact.

Reply:
I think critical is too aggressive here, I would save it for the whole install being messed up and things of that kind